web/styles/portico/portico_signin.css

```diff
@@ -1461,3 +1461,34 @@ button#register_auth_button_gitlab {
     margin-top: 5px;
     display: none;
 }
+
+#signup-button,
+#reset-button {
+    &:disabled {
+        opacity: 0.5;
+        cursor: not-allowed;
+    }
+}
+
+#reset-password-limit-indicator,
+#password-limit-indicator {
+    &:not(:empty) {
+        font-size: 16px;
+        color: var(--color-limit-indicator);
```

Comment:
Chrome browser console says that these colors are not defined. Maybe because the relevant file is not imported. You might need to copy those here.

Reply:
I am not sure about this, since previously we just copied the colors in `markdown.css`. But we can wait for Tim's feedback.